search_params.py

```diff
@@ -254,6 +259,11 @@ def main():
     else:
         scheduler = None
 
+    exp_name = '{}_{}_{}'.format(
+        config.data_name,
+        Path(config.config).stem if config.config else config.model_name,
+        datetime.now().strftime('%Y%m%d%H%M%S'),
```

Comment:
If two trials running on different GPUs start at the same time, we might get the same `exp_name`.
Would ray help handle this issue?